1.0.1/jessie/product/README.md

```diff
@@ -17,3 +17,27 @@ This image contains:
 ## Supported tags
 
 - [`1.0.1`, `latest` (*Dockerfile*)](https://github.com/aspnet/aspnet-docker/blob/master/1.0.1/jessie/product/Dockerfile)
+
+## Example Usage
+
+1. Create a Dockerfile for your application, the following example assumes you have already compiled your application (which is the expected use case for this image)
+
+  ```
+  FROM microsoft/aspnetcore
+  WORKDIR /app
+  COPY . /app
```

Comment:
I have no objection to this but I personally prefer `COPY . .` mainly because it makes changing the application directory easier since it is only specified in one place.  Since this is a sample, I would think there is a good chance people would copy and tweak it - such as change the app directory name.